args.go

```diff
@@ -241,6 +241,7 @@ func parseEditArgs(args []string) (Command, error) {
 
 	e := &Edit{}
 	e.VaultName = flag.Arg(0)
+	e.showHidden = false
```

Comment:
This is unnecessary as the "zero value" of a boolean is `false` and new instances of types initialize all fields to their zero values.